Dockerfile

```
@@ -417,6 +423,22 @@ FROM openfeature-provider-js-base AS openfeature-provider-js.build
 
 RUN make build
 
+# ==============================================================================
+# Pack OpenFeature Provider (JS) - Create tarball for publishing
+# ==============================================================================
+FROM openfeature-provider-js.build AS openfeature-provider-js.pack
+
+# Create npm package tarball
+RUN npm pack
+
+# ==============================================================================
+# Extract OpenFeature Provider (JS) package artifact
+# ==============================================================================
+FROM scratch AS openfeature-provider-js.artifact
+
+# Copy the tarball (npm pack creates spotify-confidence-openfeature-server-provider-local-*.tgz)
+COPY --from=openfeature-provider-js.pack /app/*.tgz /package.tgz
```

Change source to single package.
```suggestion
COPY --from=openfeature-provider-js.pack /app/package.tgz /package.tgz
```